lowtime/solver.py

```diff
@@ -243,13 +274,22 @@ def run(self) -> Generator[IterationResult, None, None]:
                 )
 
             try:
-                s_set, t_set = self.find_min_cut(critical_dag)
+                nodes, edges = self.format_rust_inputs(critical_dag)
+                rust_runner = _lowtime_rs.PhillipsDessouky(
+                    FP_ERROR,
+                    nodes,
+                    critical_dag.graph["source_node"],
+                    critical_dag.graph["sink_node"],
+                    edges,
+                )
+                s_set, t_set = rust_runner.find_min_cut()
```

Comment:
If you have time, make a screenshot of this diff (replacement of `self.find_min_cut` with these lines is basically all the core code change in lowtime's solver) and put it in your slides. It's extremely cool!